recipes/jpy/meta.yaml

```diff
@@ -0,0 +1,100 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "jpy" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/jpy-consortium/jpy.git 
```

Comment:
Can you use a tarball instead?

Reply:
Due to a compatibility issue with SNAP, version v0.9.0 is required. The tarball for v0.9.0 isn’t available on the repository. 
After making sure v0.9.0 is available on conda-forge, I will update it to use the tarball for newer versions.

Reply:
This is a tarball for the same code: https://github.com/jpy-consortium/jpy/archive/refs/tags/0.9.0.tar.gz

You can get the checksum for it and add it as `https://github.com/jpy-consortium/jpy/archive/refs/tags/{{ version }}.tar.gz`.